src/Helpers/ValueGetter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\PhpSnmp\Helpers;
+
+use SimPod\PhpSnmp\Exception\GeneralException;
+use SimPod\PhpSnmp\Exception\NoSuchInstanceExists;
+use SimPod\PhpSnmp\Transport\SnmpClient;
+use function array_shift;
+use function array_values;
+use function strpos;
+
+final class ValueGetter
+{
+    /**
+     * @param array<int|string, mixed> $raw
+     *
+     * @return mixed
+     *
+     * @template T
```

Comment:
should use psalm prefix

I noticed this few minutes ago https://github.com/myclabs/php-enum/commit/61c12d18b5a77fbc6d7aea67038b5bb6b801d23e

Reply:
ok